binding/form.go

```diff
@@ -68,22 +69,54 @@ type multipartRequest http.Request
 
```

Comment:
@vkd can move `multipartRequest` code to `form_mapping.go`? thanks!

Reply:
I agree with you about moving this code from `form.go` file, but I think `form_mapping.go` is not good place for that.
I prefer to move this code to new file like `multipart_form_mapping.go`. What do you think?

Reply:
@vkd OK